File: arrays/count.py
import array

arr = array.array('i',[1, 2, 5, 3, 5, 1, 2, 5])

print(arr.count(5))

# manual version
check = 5
count = 0

for i in arr:
    if check == i:
        count += 1

print(count)

File: arrays/count.c
#include <stdio.h>

int main()
{
  int i;
  int count = 0;
  int check = 5;

  int arr[] = { 1, 2, 5, 3, 5, 1, 2, 5 };

  int len = sizeof(arr)/sizeof(arr[0]);

  for (i = 0; i < len; i++)
  {
    if (check == arr[i])
    {
      count++;
    }
  }

  printf("%d\n", count);
}
